### ISSUE TITLE
Repair Checksums after whole Workflow has finished?

### ISSUE THREAD

**Issue by smogm** [michkoll/fakenevtx#3]
At the moment the checksums are recalculated after each single value modification.
Wouldn't it be more performant to first make all modifications and then recalculate all values at the end?

**Comment by michkoll** [comment]
Yes, this should be possible. Have to split the repair_header function, because the offsets in chunk header have to be fixed after each step
